spacy/lang/es/tokenizer_exceptions.py

```diff
@@ -4,22 +4,33 @@
 from ...symbols import ORTH, LEMMA, NORM, PRON_LEMMA
 
 
-_exc = {
+# Not used by default as they are only used in certains regional speech modes, left here for completeness
+_slang_exc = {
+    "pa'l": [{ORTH: "pa", LEMMA: "para"}, {ORTH: "l", LEMMA: "el", NORM: "el"}],
+    "pa'la": [{ORTH: "pa", LEMMA: "para"}, {ORTH: "la", LEMMA: "la", NORM: "la"}],
```

Comment:
You're right that "pala" shouldn't be in the normal list of exceptions. In fact, we removed it already 2 months ago, but apparently only on the `develop` branch. I probably should have done it on `master` instead. See here: https://github.com/explosion/spaCy/pull/5265/files (and the preceding discussion here: https://github.com/explosion/spaCy/issues/5255)

Reply:
Following your reasoning, I think we also just remove "pala" from the exceptions list.

Reply:
I'm sorry that I didn't check the dev branch. I do think we should remove "pal" too